test/test_icub.py

```diff
@@ -0,0 +1,116 @@
+        cls.kitchen = Object("kitchen", pycrap.Kitchen, "kitchen" + cls.extension)
+        # cls.cereal = Object("cereal", pycrap.Cereal, "breakfast_cereal.stl",
+        #                     pose=Pose([1.3, 0.7, 0.95]))
+        ProcessModule.execution_delay = False
+        ProcessModuleManager.execution_type = ExecutionType.SIMULATED
+        cls.viz_marker_publisher = VizMarkerPublisher()
+
+    def test_move_joint_direct(self):
+        self.robot.set_joint_position("l_hand_thumb_0_joint",math.radians(30))
+
+
+    def test_move_joints_simulated(self):
+        with simulated_robot:
+            MoveJointsMotion(["torso_roll"], [math.radians(-20.0)]).perform()
+            sleep(3)
+            ParkArmsActionPerformable(arm=Arms.BOTH).perform()
+            sleep(3)
+            MoveArmJointsMotion(left_arm_poses={"l_elbow": math.radians(20.0)}).perform()
+            sleep(3)
+            MoveArmJointsMotion(right_arm_poses={"r_elbow": math.radians(40.0)}).perform()
+            sleep(3)
+            LookingMotion(Pose([-2, -2, 3])).perform()
+            sleep(3)
+            MoveGripperMotion(GripperState.OPEN, Arms.RIGHT).perform()
+            MoveGripperMotion(GripperState.OPEN, Arms.LEFT).perform()
+            sleep(20)
+            MoveGripperMotion(GripperState.CLOSE, Arms.RIGHT).perform()
+            MoveGripperMotion(GripperState.CLOSE, Arms.LEFT).perform()
+            sleep(10)
+            MoveGripperMotion(GripperState.OPEN, Arms.RIGHT).perform()
+            MoveGripperMotion(GripperState.OPEN, Arms.LEFT).perform()
+
+
+
+
+
+    def test_move_joints(self):
+
+
+        with real_robot:
+            yarpModule = run_icub_state_updater(['/', '--robot', 'icubSim'])
+            if yarpModule is None:
+                print("Failed to open yarp module")
+            MoveJointsMotion(["torso_roll"], [math.radians(-20.0)]).perform()
+
+            ParkArmsActionPerformable(arm=Arms.BOTH).perform()
+
+            MoveArmJointsMotion(left_arm_poses={"l_elbow": math.radians(20.0)}).perform()
+
+            MoveArmJointsMotion(right_arm_poses={"r_elbow": math.radians(40.0)}).perform()
+            LookingMotion(Pose([-2, -2, 3])).perform()
+            MoveGripperMotion(GripperState.OPEN,Arms.LEFT).perform()
+            sleep(10)
+            MoveGripperMotion(GripperState.CLOSE,Arms.LEFT).perform()
+            MoveGripperMotion(GripperState.CLOSE, Arms.RIGHT).perform()
+            sleep(10)
+            MoveGripperMotion(GripperState.OPEN, Arms.RIGHT).perform()
+            MoveGripperMotion(GripperState.OPEN, Arms.LEFT).perform()
+            sleep(10)
+
+            if yarpModule is None:
+                print("stopping the module")
+                interrupt_and_close(yarpModule)
+
+
+    def test_park_arms(self):
+        with real_robot:
+            ParkArmsActionPerformable(arm=Arms.BOTH).perform()
+
+
+    def test_try_world(self):
+        sleep(100)
```

Comment:
What? Why?
I guess this was for testing, in this case please delete it

Reply:
same